apps/agreement/contracts/Agreement.sol

```diff
@@ -15,8 +15,6 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
 import "@aragon/os/contracts/lib/math/SafeMath64.sol";
```

Comment:
I might be missing something... but where are we importing `@aragon/staking` in the contracts and tests?

Reply:
Not sure what happened here, but it is currently imported in `implement_agreements_app` branch.